app/views/govuk_publishing_components/components/docs/chart.yml

```diff
@@ -362,4 +362,40 @@ examples:
             - 118
             - 85
             - 80
+  with_hidden_heading:
+    description: |
+      The heading element can be removed entirely from the DOM. This is needed if the heading will come from somewhere outside the component. Hiding the heading doesn't remove the need to populate `chart_heading` with text, as `chart_heading` is needed for accessibility.
```

Comment:
Could we reword this - "removed entirely from the DOM" sounds like a js interaction after load - it's actually never in the DOM to be removed. Maybe just:
The heading element is optional, allowing a heading to be set outside the component. This doesn't remove the need to populate...

Reply:
Good point, I've changed the wording.